test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersions.g.cs

```diff
@@ -19,6 +19,7 @@ namespace Datadog.Trace.ClrProfiler.IntegrationTests
     [SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1516:Elements must be separated by blank line", Justification = "This is an auto-generated file.")]
     public class PackageVersions
     {
+#if TRUE
```

Comment:
What do these `#if TRUE` accomplish?

Reply:
That was unnecessary, and I removed it in 9ac6ecda12f79f771369df2828e5520352b6ef8d